packages/squiggle-lang/src/js/cli/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { SqProject } from "../SqProject";
+import { SqValueTag } from "../SqValue";
+
+export const measure = (cb: () => void, times = 1) => {
+  const t1 = new Date();
+
+  for (let i = 1; i <= times; i++) {
+    cb();
+  }
```

Comment:
There must be some function for ``runNTimes`` in lodash or other. Or we could make one.